README.md

```diff
@@ -181,13 +181,14 @@ setLocale({
   },
 });
 
-// Now use Yup schemas AFTER you defined your custom dictionary
-const schema = yup.object().shape({
+// now use Yup schemas AFTER you defined your custom dictionary
+var schema = yup.object().shape({
```

Comment:
Let's leave the const and let calls

Reply:
Do you say just to leave this lines or to change all `var`s to `let` or `const`. 
Personally i prefer second option with `let`, i chose `var` as it was most used here before. 
If you mean just to leave, then is there any rule or a reason?

Reply:
Sorry yeah, I would perfer let everywhere 👍

Reply:
Done